executor/src/main/java/org/k8loud/executor/openstack/OpenstackProperties.java

```diff
@@ -8,13 +8,13 @@
 @Component
 @ConfigurationProperties(prefix = "openstack")
 public class OpenstackProperties {
-    private String endpoint;
-    private String username;
-    private String password;
-    private String domainName;
-    private String domainID;
-    private String projectID;
-    private String openstackAuth;
+    private String endpoint = "https://127.0.0.1:5000";
+    private String username = "username";
+    private String password = "password";
+    private String domainName = "domainName";
+    private String domainID = "domainID";
+    private String projectID = "projectID";
+    private String openstackAuth = "PROJECT_SCOPED";
```

Comment:
why

Reply:
Don't remember, I think I wanted to have default values for tests in Kube, DataStorage but ended up mocking them (which should've been done in the first place). 
I added defaults to Openstack on this occasion. Removing all defaults now